### PR TITLE
Added observatorium-dynamic datasource to dashboards

### PR DESCRIPTION
**Pull request by dislbenn**
Related issue: https://github.com/stolostron/backlog/issues/19142
Signed-off-by: dislbenn <lavontae.bennett@gmail.com>

### CODE REVIEW

**Review by marcolan018 — CHANGES_REQUESTED** [review]
we should not hardcode the datasource for each panels.
The better way is to update datasource var defined in dashboard json.
```
      "templating": {
        "list": [
          {
            "current": {
              "selected": false,
              "text": "",
              "value": ""
            },
            "description": null,
            "error": null,
            "hide": 2,
            "includeAll": false,
            "label": null,
            "multi": false,
            "name": "datasource",
            "options": [],
            "query": "prometheus",
            "refresh": 2,
            "regex": "Observatorium-Dynamic",
            "skipUrlSync": false,
            "type": "datasource"
          }
        ]
      },
```
In above datasource definition. I think add a new line " "regex": "Observatorium-Dynamic"," or similiar field can specify the datasource

**Comment by dislbenn** [comment]
> we should not hardcode the datasource for each panels. The better way is to update datasource var defined in dashboard json.
> 
> ```
>       "templating": {
>         "list": [
>           {
>             "current": {
>               "selected": false,
>               "text": "",
>               "value": ""
>             },
>             "description": null,
>             "error": null,
>             "hide": 2,
>             "includeAll": false,
>             "label": null,
>             "multi": false,
>             "name": "datasource",
>             "options": [],
>             "query": "prometheus",
>             "refresh": 2,
>             "regex": "Observatorium-Dynamic",
>             "skipUrlSync": false,
>             "type": "datasource"
>           }
>         ]
>       },
> ```
> 
> In above datasource definition. I think add a new line " "regex": "Observatorium-Dynamic"," or similiar field can specify the datasource

@marcolan018 I added the regex that contains `Observatorium-Dynamic`. Can you review if things look good?

File: operators/multiclusterobservability/manifests/base/grafana/dash-k8s-compute-resources-node-pods.yaml [hunk 1]
```diff
@@ -490,7 +507,7 @@ data:
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(container_memory_working_set_bytes{cluster=\"$cluster\", node=\"$node\", container!=\"\",job=\"kubelet\", metrics_path=\"\/metrics\/cadvisor\", image!=\"\"}) by (pod)",
+              "expr": "sum(container_memory_working_set_bytes{cluster=\"$cluster\", node=\"$node\", container!=\"\",job=\"kubelet\", metrics_path=\"/metrics/cadvisor\", image!=\"\"}) by (pod)",
```
Comment (marcolan018) [inline operators/multiclusterobservability/manifests/base/grafana/dash-k8s-compute-resources-node-pods.yaml:510]: why modify expr here?

**Comment by marcolan018** [comment]
the changes for datasource looks good for me, but better to verify that in your env with 2nd datasource available.

My concern is why the pr includes some changes on the expr in the dashboard json?

@dislbenn

**Comment by dislbenn** [comment]
> the changes for datasource looks good for me, but better to verify that in your env with 2nd datasource available.
> 
> My concern is why the pr includes some changes on the expr in the dashboard json?
> 
> @dislbenn

It seems that those changes were made automatically when the dashboard was updated. Let me revert those expressions.

File: operators/multiclusterobservability/manifests/base/grafana/dash-k8s-compute-resources-node-pods.yaml (hunk 1 above)
Comment (dislbenn) [reply operators/multiclusterobservability/manifests/base/grafana/dash-k8s-compute-resources-node-pods.yaml:510]: This seems to be made when I updated the dashboard. Let me take a look at it.

**Comment by dislbenn** [comment]
@marcolan018 expression changes have been reverted.

**Pull request merged**